src/LibraryManager.Contracts/CompletionSet.cs

```diff
@@ -53,4 +58,17 @@ public struct CompletionItem
         /// </summary>
         public string Description;
     }
+
+    /// <summary>
+    /// The completion sort order is to indicate the sorting type of the completion item.
+    /// </summary>
+    public enum CompletionSortOrder
+    {
+        /// <summary>The completion item is alphabetical sorted.</summary>
+        Alphavetical,
```

Comment:
typo

Reply:
Will change.

---
In reply to: [205555312](https://github.com/aspnet/LibraryManager/pull/312#discussion_r205555312) [](ancestors = 205555312)